controllers/users.js

```diff
@@ -301,7 +335,9 @@ class Users {
    * @returns {object} res
    */
   static async getAllAuthors(req, res) {
-    const result = await User.findAll({ attributes: ['image', 'username', 'email'] });
+    const result = await User.findAll({
+      attributes: ['image', 'username', 'email']
+    });
```

Comment:
@shemaeric as when you delete a user you change activated to **-1** When getting a user, add a condition for it so that we don't return deleted users.